KSystemInformer/protection.c

```diff
@@ -186,14 +186,14 @@ BOOLEAN KphpShouldSuppressObjectProtections(
 
     status = KphDominationCheck(Target->EProcess, Actor->EProcess, UserMode);
     if (!NT_SUCCESS(status))
-    {
-        //
-        // If the process being protected doesn't dominate the target then
-        // do not strip any access from it.
+    {
+        //
+        // Grant access when the actor is a protected process and the target is

```

Comment:
> I noticed that the comment in KphpShouldSuppressObjectProtections seams to have been completely wrong so I made a new one.

The comment and trace seems to read fine to me, what's the problem with leaving it?

Reply:
The original comment says ´´´If the process being protected doesn't dominate the target then do not strip any access from it´´´
this means if booth processes have for example ```Protection.Type == PsProtectedTypeNone``` than no access will be stripped. But this is not what happens in such a scenario rightfully access is being stripped, as required for the protection to work.
What the code does is it allows any PPL protected process to access an KSI protected process.
So the original comment does not correctly reflect whats actually happening.

Reply:
Okay 👍